packages/fuel-gauge/src/predicate/utils/predicate/fundPredicate.ts

```diff
@@ -1,9 +1,12 @@
 import type { InputValue, BigNumberish, WalletUnlocked, Predicate } from 'fuels';
 import { ScriptTransactionRequest, BN } from 'fuels';
 
-export const fundPredicate = async <T extends InputValue[]>(
+export const fundPredicate = async <
+  T extends InputValue[] = InputValue[],
+  C extends { [name: string]: unknown } | undefined = { [name: string]: unknown },
+>(
```

Comment:
```suggestion
export const fundPredicate = async (
```

Reply:
without this change lint is not able to pass